lib/ramble/ramble/spack_runner.py

```diff
@@ -345,6 +351,28 @@ def get_package_path(self, package_spec):
             self._dry_run_print(args)
             return '/dry-run/path/to/%s' % package_spec.split()[0]
```

Comment:
Is it worth changing this to `os.path` for portability? (even though it's not a real path...)